2022-05-21 12:23:10.803000: E tensorflow/stream_executor/cuda/cuda_driver.cc:271] failed call to cuInit: CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
2022-05-21 12:23:11.105498: E tensorflow/stream_executor/cuda/cuda_driver.cc:271] failed call to cuInit: CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
2022-05-21 12:23:11.787427: E tensorflow/stream_executor/cuda/cuda_driver.cc:271] failed call to cuInit: CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
2022-05-21 12:23:12.030872: E tensorflow/stream_executor/cuda/cuda_driver.cc:271] failed call to cuInit: CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
2022-05-21 12:23:12.105891: E tensorflow/stream_executor/cuda/cuda_driver.cc:271] failed call to cuInit: CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
2022-05-21 12:23:12.346581: E tensorflow/stream_executor/cuda/cuda_driver.cc:271] failed call to cuInit: CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
2022-05-21 12:23:12.429996: E tensorflow/stream_executor/cuda/cuda_driver.cc:271] failed call to cuInit: CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
2022-05-21 12:23:12.741345: E tensorflow/stream_executor/cuda/cuda_driver.cc:271] failed call to cuInit: CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
2022-05-21 12:23:12.950296: E tensorflow/stream_executor/cuda/cuda_driver.cc:271] failed call to cuInit: CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
2022-05-21 12:23:13.042283: E tensorflow/stream_executor/cuda/cuda_driver.cc:271] failed call to cuInit: CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
2022-05-21 12:23:13.096719: E tensorflow/stream_executor/cuda/cuda_driver.cc:271] failed call to cuInit: CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
2022-05-21 12:23:13.182649: E tensorflow/stream_executor/cuda/cuda_driver.cc:271] failed call to cuInit: CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
2022-05-21 12:24:33.032154: E tensorflow/stream_executor/cuda/cuda_driver.cc:271] failed call to cuInit: CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
File "./theories/Reals/Rpower.v", line 168, characters 2-47:
Error:
Anomaly
"Uncaught exception Tactician_reinforce_plugin.Neural_learner.NeuralLearner(TS).MismatchedArguments."
Please report at http://coq.inria.fr/bugs/.

File "./theories/Reals/RiemannInt_SF.v", line 244, characters 2-94:
Error: Anomaly "Capnp protocol error 3b"
Please report at http://coq.inria.fr/bugs/.

File "./theories/Reals/RiemannInt_SF.v", line 269, characters 2-22:
Error:
Anomaly "Uncaught exception Unix.Unix_error(Unix.EPIPE, "single_write", "")."
Please report at http://coq.inria.fr/bugs/.

File "./theories/Reals/RiemannInt_SF.v", line 298, characters 2-443:
Error:
Anomaly "Uncaught exception Unix.Unix_error(Unix.EPIPE, "single_write", "")."
Please report at http://coq.inria.fr/bugs/.

File "./theories/Reals/RiemannInt_SF.v", line 316, characters 2-21:
Error:
Anomaly "Uncaught exception Unix.Unix_error(Unix.EPIPE, "single_write", "")."
Please report at http://coq.inria.fr/bugs/.

